PyDSS/cli/reports.py

```diff
@@ -0,0 +1,139 @@
+"""
+CLI to run a PyDSS project
+"""
+
+import ast
+import logging
+import os
+import sys
+import json
+import click
+
+from PyDSS.pydss_project import PyDssProject
+from PyDSS.loggers import setup_logging
+from PyDSS.utils.utils import get_cli_string, make_human_readable_size
+from PyDSS.common import SIMULATION_SETTINGS_FILENAME
+from terminaltables import SingleTable
+
+logger = logging.getLogger(__name__)
+
+@click.argument(
+    "project-path",
+)
+
+@click.option(
+    "-l", "--list",
```

Comment:
```suggestion
    "-l", "--list", "list_"
```
If you make this change then your function will accept the variable `list_` and you won't override the python type `list`.

Reply:
fixed